README.md

```diff
@@ -5,31 +5,32 @@
 This plugin provides syntax highlighting and auto-completion support for
 PostgreSQL version 9.6 or above and for some of its extensions, including:
 
-- PL/pgSQL;
-- [PostGIS](http://postgis.net) (including PostGIS Topology);
-- [pgRouting](http://pgrouting.org);
-- [pgTap](http://pgtap.org).
+-   PL/pgSQL;
+-   [PostGIS](http://postgis.net) (including PostGIS Topology);
+-   [pgRouting](http://pgrouting.org);
+-   [pgTap](http://pgtap.org).
 
```

Comment:
Can you direct me to the CommonMark spec for this change?

Reply:
When I run `prettier` on the current Readme and (empty) `.prettierrc`, it reverts this change. Was this change intentional?